docs/source/index.rst

```diff
@@ -45,3 +45,10 @@
 EinsteinPy is an open source pure Python package dedicated to problems arising in General Relativity and relativistic physics, such as goedesics plotting for schwartzschild space-time model, calculation of schwartzschild radius for any mass given. Features like visualisation of geodesics of curved black holes and 3D visualisations are some of the features which are planned.
 It is released under the MIT license.
 
+Contents
+--------
+
+.. toctree::
```

Comment:
Its great that you figured out the `toctree`

Reply:
made bhavya co-author of the commit. Merge after the tests get passed

Reply:
@shreyasbapat travis build failed due to their connection error.